lib/gui/components/safe-webview.js

```diff
@@ -46,6 +46,22 @@ const VERSION_PARAM = 'etcher-version';
 const ELECTRON_SESSION = 'persist:success-banner';
 
 /**
+ * @summary API version search parameter key
+ * @constant
+ * @private
+ * @type {String}
+ */
+const API_VERSION_PARAM = 'api-version';
```

Comment:
Can we add a big comment in the `@description` section stating that changing this will break compatibility with the HTML banner and therefore we should only change it if absolutely necessary?